### PR TITLE
EUI-4100 task check at start multiple tasks

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4100

### Change description ###
Event to be performed has >1 associated tasks, 1 or more of the associated tasks are assigned to me, and I have started the event from Tasks tab for one of the associated tasks

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/shared/components/event-management/services/event-state-machine.service.ts [hunk 1]
```diff
@@ -164,6 +173,15 @@ export class EventStateMachineService {
   public entryActionForStateOneTaskAssignedToUser(state: State, context: StateMachineContext): void {
     // Trigger final state to complete processing of state machine
     state.trigger(StateMachineStates.FINAL);
+
+    // Get task assigned to user
+    let task = context.tasks.find(x => x.id === context.taskId);
+    if (task === undefined) {
```
Comment (udaydenduluri33) [inline src/shared/components/event-management/services/event-state-machine.service.ts:179]:
```suggestion
    if (!task) {
```

**Review by udaydenduluri33 — APPROVED** [review]
@johnbenjamin-hmcts  LGTM

File: src/shared/components/event-management/services/event-state-machine.service.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/shared/components/event-management/services/event-state-machine.service.ts]: Thank you @udaydenduluri33, I have now done the changes as you suggested.

**Review by kuda1992 — APPROVED** [review]
Looks good to me

**Pull request merged**